src-self-hosted/ir.zig

```diff
@@ -243,16 +243,10 @@ pub const Inst = struct {
             return 2;
         }
         pub fn getOperand(self: *const BinOp, index: usize) ?*Inst {
-            var i = index;
-
-            if (i < 1)
+            if (index == 0)
                 return self.lhs;
-            i -= 1;
-
-            if (i < 1)
+            if (index == 1)
                 return self.rhs;
-            i -= 1;
```

Comment:
Hmm, this is interesting. Any idea why these were here in the first place? They seem to be no-op wrt to the calling arguments.

Reply:
These are following the same pattern used in iterating AST nodes. Probably intended to reduce errors from typos and such.

Reply:
Yes that's right